### PR TITLE
Add config to enable dynamic quantize

### PR DESCRIPTION
**Pull request by byungilm**
+ Config to set dynamic_quantization_group_size for ov

### CODE REVIEW

File: llm_bench/python/benchmark.py [hunk 1]
```diff
@@ -674,6 +674,15 @@ def get_argprser():
         'if the value is False (default), input prompts are processed in interleave manner'
     )
     parser.add_argument('-od', '--output_dir', help='Save the input text and generated text, images to files')
+    parser.add_argument(
+        '-dq',
+        '--dynamic_quantization_group_size',
+        default=None,
+        required=False,
+        type=int,
+        help='Set group size of dynamic quantization. Dynamic quantization for GPU is enabled if this argument is equal to or bigger than 32.',
```
Comment (isanghao) [inline llm_bench/python/benchmark.py:683]: "Set group size of FC dynamic quantization" would be enough

Comment (byungilm) [reply llm_bench/python/benchmark.py]: Fixed

**Review by byungilm — COMMENTED** [review]
Mingyu

File: llm_bench/python/benchmark.py [hunk 2]
```diff
@@ -674,6 +674,15 @@ def get_argprser():
         'if the value is False (default), input prompts are processed in interleave manner'
     )
     parser.add_argument('-od', '--output_dir', help='Save the input text and generated text, images to files')
+    parser.add_argument(
```
Comment (wgzintel) [inline llm_bench/python/benchmark.py:677]: Do we need to check the value of dynamic_quantization_group_size? Can this value be less than or equal to 0?

Comment (byungilm) [reply llm_bench/python/benchmark.py:677]: This value can be equal to 0 which means disable dynamic quantizing.

**Comment by byungilm** [comment]
Use '-lc config.json' 
The file config.json contains :
config =  {"DYNAMIC_QUANTIZATION_GROUP_SIZE":"64"}

**Pull request closed without merging**